inc/class-sitemaps-index.php

```diff
@@ -27,7 +34,8 @@ public function bootstrap() {
 	 * Sets up rewrite rule for sitemap_index.
 	 */
 	public function register_sitemap() {
-		$this->registry->add_sitemap( 'sitemap_index', 'sitemap\.xml$' );
+		$post_type = 'index';
```

Comment:
This line is unused now we're using post_type.
 I think we might need a task to separate $name from $post_type as there is no custom post type called index, but for the code that we have it's good.

Reply:
yeah, sorry I updated this and removed it as you were reviewing 🤦‍♀